sor/src/main/java/com/bazaarvoice/emodb/sor/core/DefaultCompactor.java

```diff
@@ -28,19 +30,19 @@ public DefaultCompactor(Counter archiveDeltaSizeInMemory, boolean keepDeltaHisto
     }
 
     public Expanded doExpand(Record record, long fullConsistencyTimestamp, MutableIntrinsics intrinsics, boolean ignoreRecent,
-                                Map.Entry<UUID, Compaction> compactionEntry)
+                                Map.Entry<DeltaClusteringKey, Compaction> compactionEntry)
             throws RestartException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.bazaarvoice.emodb.sor.core.AbstractCompactor$RestartException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)